www/components/Docs/SymbolLinkTable.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box } from "@welcome-ui/box";
+import { Link } from "@welcome-ui/link";
+import { Text } from "@welcome-ui/text";
+import NextLink from "next/link";
+import React from "react";
+import { UrlObject } from "url";
+
+import Table from "@/components/CondensedTable";
+
+import { sortedBy } from "@/lib/utils";
+
+import Documentation from "./Documentation";
+
+interface Sym {
+  name: string;
+  documentation: string[];
+}
+
+interface Props<T extends Sym> {
+  title: string;
+  symbols: T[];
+  url: (sym: T) => string | UrlObject;
+}
+
+function deduplicate<T extends Sym>(xs: T[]): T[] {
+  return sortedBy(xs, "name", "documentation")
+    .reverse()
+    .filter((x, i, xs) => {
+      return i === 0 || x.name !== xs[i - 1].name;
+    })
+    .reverse();
+}
```

Comment:
I feel ambivalent about this. Apart from `@overload` this shouldn't be necessary especially after #19 
For `@overload`s, there will need to be some special handling: we don't want to drop overloads as they meaningfully help understand the API (probably more than the signature of the implementation).

Reply:
I implemented this specifically for functions. Agree that once (if) we figure out a way to uniquely represent each signature, this should go away. That needs some care - for example, `click` only adds docstrings on one of the versions.